MAUI/DataGrid/Column-sizing.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: post
+title: Column Sizing in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about Column Sizing support in Syncfusion MAUI DataGrid (SfDataGrid) control and more.
```

Comment:
description: Learn all about Column Sizing support in Syncfusion MAUI DataGrid (SfDataGrid) control and more here.